### PR TITLE
Fix/rules json

### PR DESCRIPTION
**Pull request by MiniMitre**
Copied over all rule descriptions from the V4.0 game manual pdf and added to rules.json

Also, created seperate rules lists for VRC, VAIC, and VEXU. This makes the list shorter for VRC, and VEXU competitions, which makes it easier for referees to find the rule they want, by only showing rules relevant to each particular game.

Feedback wanted on the ordering of the rules. I do not think the order of the quick reference guide is the best, I have settled on:

[\<VAIG>]
[\<VAIT>]
[\<VAIR>]
(\<VUG>)
\<SG>
\<G>
(\<VUR>)
\<R>
\<S>
\<SC>
(\<VURS>)
\<RS>
(\<VUT>)
\<T>

This has required duplication, but works for now and can be fixed 'properly' next season

### CODE REVIEW

File: public/rules.json [hunk 1]
```diff
@@ -35,148 +192,192 @@
...
               "rule": "<G7>",
-              "description": "Don’t clamp your Robot to the field",
+              "description": " Don’t clamp your Robot to the Field",
               "link": "https://www.robotevents.com/storage/game_manual/VRC_2023-2024_Over_Under/rules/G7.html"
             },
+
             {
               "rule": "<G8>",
-              "description": "Only Drivers, and only in the Alliance Station",
+              "description": " Only Drive Team Members, and only in the Alliance Station",
```
Comment (brenapp) [inline public/rules.json:237]: You should get rid of the leading space in every rule description. Is that to space it out in the Rules selector?

**Review by brenapp — CHANGES_REQUESTED** [review]

File: test/bulk/.env [hunk 2]
```diff
@@ -1,3 +1,4 @@
 SKU=RE-VRC-23-1488
 CODE=C0F-BEF
-DIVISION=1
\ No newline at end of file
+DIVISION=1
+VITE_REFEREE_FYI_BUILD_MODE=WC
```
Comment (brenapp) [inline test/bulk/.env:4]: Sorry I should have been more clear. The root directory of the project, you should make a .env file (it's gitignored) to include that setting

File: public/rules.json [hunk 3]
```diff
@@ -521,200 +931,1395 @@
...
           ]
-        },
+        }
+      ]
+    },
+    {
+      "title": "Full Volume",
+      "season": "2023-2024",
+      "programs": ["VIQRC"],
+      "ruleGroups": [
```
Comment (brenapp) [inline public/rules.json:2322]: Should we make the IQ rule order consistent with the other programs?

**Review by brenapp — CHANGES_REQUESTED** [review]

**Comment by MiniMitre** [comment]
Grouped together similar rules, so there are fewer sections in total. Decided on the following ordering based on the general likelihood of violations occuring.

Specific Game Rules
\<SG>

General / Safety Rules (
[\<VAIG>]
(\<VUG>)
\<G>
\<S>

Robot Rules
[\<VAIR>]
(\<VUR>)
\<R>

Scoring/Skills Rules
\<SC>
(\<VURS>)
\<RSC>

Tournament Rules
[\<VAIT>]
(\<VUT>)
\<T>

\<SC1> for IQ is a special case which is more likely to come up than usual for a scoring rule. However, for consistency the ordering is kept the same as VRC+

**Review by brenapp — APPROVED** [review]

**Pull request merged**